charts/newrelic-pixie/values.yaml

```diff
@@ -0,0 +1,30 @@
+# IMPORTANT: The Kubernetes cluster name
+# https://docs.newrelic.com/docs/kubernetes-monitoring-integration
+# cluster: ""
```

Comment:
in the readme these values are documented with the `global.`. I would try do document both, or provide an example for both in the values.yaml

For example in https://github.com/newrelic/helm-charts/tree/master/charts/nri-kube-events it is done in that way